integration_tests/features/support/steps.js

```diff
@@ -546,14 +585,18 @@ Then(/all wallets detect all transactions are at least Completed/, {timeout: 120
         var walletInfo = await walletClient.identify();
 
         let txIds = this.transactionsMap.get(walletInfo["public_key"]);
-        console.log("\nDetecting transactions as at least Completed: ", walletName, txIds)
+        if (txIds === undefined) {
+            console.log("\nNo transactions for " + walletName + "!")
+            expect(false).to.equal(true);
+        }
+        console.log("\nDetecting", txIds.length, "transactions as at least Completed: ", walletName, txIds)
         for (i = 0; i < txIds.length; i++) {
             // Get details
-            console.log("\n" + wallet.name + ": Waiting for transaction " + txIds[i] + " to register at least Completed in the wallet ...");
-            await waitFor(async() => wallet.getClient().isTransactionAtLeastCompleted(txIds[i]), true, 1100*1000, 5*1000, 5);
+            console.log("(" + (i + 1) + "/" + txIds.length + ") - " + wallet.name + ": Waiting for TxId:" + txIds[i] + " to register at least Completed in the wallet ...");
+                                                                                                                           await waitFor(async() => wallet.getClient().isTransactionAtLeastCompleted(txIds[i]), true, 1100*1000, 5*1000, 5);
```

Comment:
This got pushed out to the right somehow, must have been me with the PR I submitted to your branch:
```suggestion
            await waitFor(async() => wallet.getClient().isTransactionAtLeastCompleted(txIds[i]), true, 1100*1000, 5*1000, 5);
```